packages/app/obojobo-document-engine/src/scripts/viewer/components/nav.js

```diff
@@ -189,7 +190,13 @@ export default class Nav extends React.Component {
 		return <div className="lock-icon" />
 	}
 
+	onClickRedAlert() {
+		const isRedAlertEnabled = NavUtil.isRedAlertEnabled(this.props.navState)
+		NavUtil.setRedAlert(!isRedAlertEnabled)
+	}
+
 	render() {
+		const isRedAlertEnabled = NavUtil.isRedAlertEnabled(navState)
```

Comment:
This line references `navState` but it's not set until the line below. Once I moved this line down everything worked as expected.